modules/account-lib/src/coin/stx/transferBuilder.ts

```diff
@@ -0,0 +1,103 @@
+import { BaseCoin as CoinConfig } from '@bitgo/statics';
```

Comment:
All of this needs to be separated back into the transactins & transfer builder PR's. Need to clean this up